src/form-question-types.rst

```diff
@@ -1658,33 +1659,151 @@ In the parameters column, write ``max-pixels=`` followed by the desired maximum
 Audio widget
 ----------------
 
-Records audio from an external app.
+type
+  :tc:`audio`
+appearance
+  :tc:`none`
 
-.. note::
+Records audio using the device's microphone or a connected external microphone. By default, an :ref:`external application <external-audio-app>` is used. Starting with Collect v1.29, you can also use :ref:`built-in recording <built-in-audio-recording>`.
 
-  Some Android devices provide a default application for audio recording.
-  Others do not,
-  and the user will need to install an audio recording app.
+.. image:: /img/form-widgets/audio-start.*
```

Comment:
I took the container out from around this screenshot as the whitespace wasn't behaving correctly when it was minimized (very little space between it and the rubric below). It's also the only widget with a minimized screenshot as far as I could see? Happy for it to go back in if there are strong feelings about it.

Reply:
It just doesn't feel all that useful to me but let's leave it as-is for now and we can always tweak!